Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -100,12 +111,31 @@ final class PledgeCTAContainerView: UIView {
 
 // MARK: - Styles
 
-private let projectStateButtonStyle: ButtonStyle = { (button: UIButton) in
-  button
-    |> roundedStyle(cornerRadius: 12)
-    |> UIButton.lens.titleColor(for: .normal) .~ .white
-    |> UIButton.lens.titleLabel.font .~ .ksr_headline(size: 15)
-    |> UIButton.lens.layer.borderWidth .~ 0
-    |> UIButton.lens.titleEdgeInsets .~ .init(topBottom: Styles.grid(1), leftRight: Styles.grid(2))
-    |> (UIButton.lens.titleLabel .. UILabel.lens.lineBreakMode) .~ .byWordWrapping
+private func adaptableStackViewStyle(_ isAccessibilityCategory: Bool) -> (StackViewStyle) {
+  return { (stackView: UIStackView) in
+    let axis: NSLayoutConstraint.Axis = (isAccessibilityCategory ? .vertical : .horizontal)
+    let spacing: CGFloat = (isAccessibilityCategory ? Styles.grid(1) : 0)
+
+    return stackView
+      |> \.axis .~ axis
+      |> \.spacing .~ spacing
+  }
+}
+
+private func projectStateButtonStyle(
+  _ isAccessibilityCategory: Bool, amountAndRewardTitleStackViewIsHidden: Bool
+) -> (ButtonStyle) {
+  return { (button: UIButton) in
+    let lineBreakMode: NSLineBreakMode = isAccessibilityCategory || amountAndRewardTitleStackViewIsHidden
+      ? NSLineBreakMode.byWordWrapping : NSLineBreakMode.byTruncatingTail
+
+    return button
+      |> roundedStyle(cornerRadius: 12)
+      |> UIButton.lens.titleColor(for: .normal) .~ UIColor.white
+      |> UIButton.lens.titleLabel.font .~ UIFont.ksr_headline(size: 15)
+      |> UIButton.lens.layer.borderWidth .~ 0
+      |> UIButton.lens.titleEdgeInsets .~ .init(topBottom: Styles.grid(1), leftRight: Styles.grid(2))
+      |> (UIButton.lens.titleLabel .. UILabel.lens.textAlignment) .~ NSTextAlignment.center
```

Comment:
| Before 🐛 | After 🦋 |
| --- | --- |
| <img width="460" alt="Screen Shot 2019-06-25 at 5 48 36 PM" src="https://user-images.githubusercontent.com/387596/60144123-1a9de000-9776-11e9-93d2-42b96579201a.png"> | <img width="460" alt="Screen Shot 2019-06-25 at 5 48 36 PM" src="https://user-images.githubusercontent.com/387596/60144129-238eb180-9776-11e9-9787-193c7d844ef7.png"> |

Fixes the centering